### DISCUSSION TITLE
网站新建php环境时有一个ftp账号密码，这个在哪里找到？设置后忘记保存了

### DISCUSSION THREAD

**Discussion by MemoryCollection** [1Panel-dev/1Panel#7542] [Q&A]
网站新建php环境时有一个ftp账号密码，这个在哪里找到？设置后忘记保存了

**Comment by wanghe-fit2cloud** [comment]
FTP 用户信息会同步到 `工具箱` - `FTP` 列表，如图：
![image](https://github.com/user-attachments/assets/feee4b85-a699-4191-9f86-468093b2128b)
